src/configure.in

```diff
@@ -807,6 +807,8 @@ then
     AC_MSG_ERROR([awk not found])
 fi
 
+if test $RTS != "uspace"
+then
```

Comment:
I think this breaks everything on linux uspace :frowning: 
halrun fails like so:
~~~~
/home/jepler/src/linuxcnc/trasz/scripts/realtime: line 116: syntax error near unexpected token `|'
/home/jepler/src/linuxcnc/trasz/scripts/realtime: line 116: `            if  | awk '{print $1}' | grep -x $MOD >/dev/null ; then'
~~~~